terraform/variables/terraform/tfc/variables.tf

```
@@ -0,0 +1,15 @@
+variable "tfc_org_name" {
+  description = "Name of your desired tfc org. Must be unique."
+  type        = string
+}
+
+variable "tfc_token" {
+  description = "The token used to authenticate with Terraform Cloud"
```

Super minor, but I remember terraform complaining to me in the past for not having a capital letter to start the sentence and a period to end it.
```suggestion
  description = "The token used to authenticate with Terraform Cloud."
```